testfm/constants.py

```diff
@@ -1,17 +1,25 @@
-import configparser
+from dynaconf import Dynaconf
 
-config = configparser.ConfigParser()
-config.read("testfm.properties")
+settings = Dynaconf(
+    envvar_prefix="TESTFM",
+    core_loaders=["YAML"],
+    settings_file=["testfm.yaml"],
+    preload=["conf/*.yaml"],
+    includes=["testfm.sample.yaml", ".secrets.yaml", ".secrets_*.yaml"],
```

Comment:
Do you want to include the sample as well?

Reply:
trying to keep this settings structure as similar to robottelo, so yes.

Reply:
This is what we have on robottelo. We're including local settings, instead of sample settings.

![Screenshot from 2021-03-12 14-56-29](https://user-images.githubusercontent.com/6618303/110991899-3595f680-8343-11eb-9c05-77dc23f14d65.png)

Reply:
@JacobCallahan Gotcha, good catch, updated settings to include local settings.